src/types.ts

```diff
@@ -183,9 +183,9 @@ export interface ModulePublicRuntimeConfig {
 }
 
 export declare interface Session {
-  id: string;
-  createdAt: Date;
-  ip?: string;
+  readonly id: string;
+  readonly createdAt: Date;
+  readonly ip?: string;
```

Comment:
and, sorry to kind of push this in here, but as we're already working on the `Session` interface: Would you consider adapting the `h3` event context as done here:
https://github.com/sidebase/nuxt-prisma/blob/b1ffd717cf1061215498fa8e9b90ce62cac8b494/src/runtime/server/middleware/prisma.ts#L7-L11

whuile you're add it? This way, typing for the end-user should improve when they access `event.context.session`

Reply:
Added it, but somehow the types aren't available when packed and imported as module in a project. Any suggestions?